apps/builddao/widget/components/modals/VisibilityModal.jsx

```diff
@@ -0,0 +1,39 @@
+const Overlay = styled.div`
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  display: grid;
+  place-items: center;
+  overflow-y: auto;
+  z-index: 1000;
+  width: 100vw;
+  height: 100vh;
+  background: var(--modal-overlay-color, rgba(35, 36, 43, 0.7));
+`;
+
+function VisibilityModal({
+  children,
+  title,
```

Comment:
why do we have this?
we already have a `Modal` component right?

Reply:
Yes. Utilized the Modal component now.